tests/qat/test_quantum_backend.py

```diff
@@ -632,3 +638,28 @@ def test_execution_batching(self, repeat_count, repeat_limit, expected_batches):
         engine = get_test_execution_engine(hw)
         generated_batches = engine._generate_repeat_batches(repeat_count)
         assert generated_batches == expected_batches
+
+    def test_empty_channels_removed(self):
+        hw = get_test_model()
+        qubit = hw.get_qubit(0)
+
+        builder = (
+            get_builder(hw).X(qubit, np.pi / 2.0).measure_mean_z(qubit).synchronize(qubit)
+        )
```

Comment:
not enough testing, add more builders that involve for example a variety of pulse channels: drive, measure, acquire, second state, ...

Reply:
@willh727-OQC

Reply:
They are all added when you add a qubit! Do I need more tests for just check these channels?